mhkit/river/resource.py

```diff
@@ -1,11 +1,24 @@
+"""
+resource.py
+
+Computes resource assessment metrics, including exceedance probability,
+inflow velocity, and power (theoretical resource). Calculations are based
+on IEC TS 62600-301:2019 ED1.
+
+"""
```

Comment:
Noting below the equivalent river.resource module docstring currently in the documentation.  The new description looks good to me.

"The resource submodule uses discharge data to compute exeedance probability, velocity, and power. The module also contains functions to compute the Froude number and to fit a polynomial to a series of points. The polynomial is used to estimate the relationship between discharge and velocity or velocity and power at an individual turbine."